src/components/Dashboard/FilingHistoryList/LimitedRestorationFiling.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div v-if="filing">
+    <h4>Limited Restoration Period</h4>
+
+    <p class="limited-restoration-period">
+      The Company <strong>{{ filing.legalName }}</strong> was successfully
+      restored and active <strong>until {{ expiryDateFriendly }}</strong>
```

Comment:
Should be "is active" ?

Should have period at end of sentence?

Reply:
Good catch on the missing period.  I didn't add "is" because that's not what was requested, but I'm happy to add it anytime.
![image](https://user-images.githubusercontent.com/25233684/214984389-4ee29e3d-ae82-4427-aefa-3914d7ee065d.png)

Reply:
Double-check with Ethan. To me, it doesn't read right otherwise.

Reply:
@ethantspitt can you weigh in on this thread?  I'm happy to change it however you wish.

Reply:
Thanks for checking! let's use "is active"

Reply:
Thanks Ethan.  Will do.